qendpoint-store/src/test/resources/queryplan10.txt

```diff
@@ -0,0 +1,123 @@
+Distinct (resultSizeActual=12, totalTimeActual=9.8s, selfTimeActual=0.048ms)
```

Comment:
I managed to manually optimize the query and it runs in about 10 seconds on my laptop.

Reply:
what did you change?

Reply:
ah, you mean, you still have the problem that the query planner is not woriking and you tried to reorder the triple patterns by hand only?

Reply:
Yeah. I managed to reorder the query by hand.